### PR TITLE
fix(scripts/ZulGurub): Add "Whirlwind" - High Priestess Arlokk

### PR DESCRIPTION
**Pull request by Si1ker**
## Changes Proposed:
-  Add "Whirlwind"

## SOURCE:

https://classic.warcraftlogs.com/reports/z7dxmQW2RBNC9br4#fight=6&hostility=1&type=damage-done&source=42&ability=15589

https://www.wowhead.com/classic/guide/high-priestess-arlokk-zul-gurub-strategy

## Tests Performed:

- Tested in-game

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
The Whirlwind is fine but I saw something strange with Mark of Arlokk.
The first cycle, Two had the Mark, and all adds attacked them. When the Mark of Arlokk target switched to One, the Prowlers didn't switch target as well, instead they stayed on Two, occasionally turning to face One, but turning away just as fast.
The Prowlers slowly changed to the new target though, one at a time. When it was time to cast Mark of Arlokk again, about half the Prowlers were attacking the correct target, while the other half was still attacking the old one.

image

**Comment by Si1ker** [comment]
@Gultask updated.

**Comment by Si1ker** [comment]
@Gultask pump

**Comment by Si1ker** [comment]
@Gultask 🤔

**Comment by Gultask** [comment]
Sorry. Low amount of testers + too much on my plate at the moment.

**Comment by Si1ker** [comment]
> Sorry. Low amount of testers + too much on my plate at the moment.

@Gultask It's not to test but approve, I removed buggy mark behavior, and that's it, it is ready.

**Comment by Si1ker** [comment]
cc @FrancescoBorzi

**Comment by FrancescoBorzi** [comment]
@Si1ker why closing this?

File: src/server/scripts/EasternKingdoms/ZulGurub/boss_arlokk.cpp [hunk 1]
```diff
@@ -15,15 +15,6 @@
...
  */
 
-/* ScriptData
-TCName: Boss_Arlokk
-TC%Complete: 95
-TCComment: Wrong cleave and red aura is missing not yet added.
-TCComment: Prowlers moving through wall hopefully mmaps will fix.
-TCComment: Can't test LOS until mmaps.
-TCCategory: Zul'Gurub
-EndScriptData */
```
Comment (acidmanifesto) [inline src/server/scripts/EasternKingdoms/ZulGurub/boss_arlokk.cpp:25]: is there a reason why this is being removed?

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Pull request closed without merging**